.travis.yml

```diff
@@ -16,54 +15,19 @@ branches:
     - /^stable\d*$/
 
 before_install:
-  - export DISPLAY=:99.0
-  - export CHROME_BIN=chromium-browser
-  - sh -e /etc/init.d/xvfb start
-  - sudo apt-get update
-  - sudo apt-get install firefox
-  - wget https://raw.githubusercontent.com/owncloud/administration/master/travis-ci/before_install.sh
+  - wget https://raw.githubusercontent.com/nextcloud/travis_ci/master/before_install.sh
   - bash ./before_install.sh $APP_NAME $CORE_BRANCH $DB
```

Comment:
Use ` - . ./before_install.sh $APP_NAME $CORE_BRANCH $DB` (replacing `bash` with `.`) to fix unit tests on php 7.0 and 7.1